### PR TITLE
Remove Grammatical Errors

### PR DESCRIPTION
**Pull request by rishav142k**
Remove simple grammatical errors.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Naming things is hard, this part is very important! Best practice: name the proj
 
 While most of our `READMEs` should have some common features, different repo types have different requirements. Here is what we expect from the different types of repo.
 
-| Library (e.g NodeJS lib)                                        | Demo/Tool (e.g. Demo for booth/customer use)                    | Supporting code (e.g. for blog post)          |
```
Comment (lukeocodes) [inline README.md:34]: booth is not a spelling mistake :) 

@rishav142k if you can edit the PR I will consider merging it.

**Review by lukeocodes — COMMENTED** [review]
Thanks for your contribution! :)

**Comment by rishav142k** [comment]
Changed both back to booth. Is it okay, now?

File: README.md (hunk 1 above)
Comment (rishav142k) [reply README.md:34]: Yes, sure. Thanks for the suggestion.

**Review by lukeocodes — APPROVED** [review]
thanks for your contribution

**Pull request merged**